src/main/java/frc/robot/commands/AutoPickupNote.java

```diff
@@ -0,0 +1,124 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+import frc.robot.AutoRotateUtil;
+import edu.wpi.first.math.controller.PIDController;
+import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.SkyLimelight;
+import frc.robot.Constants;
+import frc.robot.subsystems.IntakeSubsystem;
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+
+public class AutoPickupNote extends Command{
+    public SkyLimelight limelight;
+    public Swerve swerve;
+    public IntakeSubsystem intakeSubsystem;
+    public Debouncer debouncer;
+    public Rotation2d yaw;
+    private AutoRotateUtil autoUtil;
+
+    public PIDController yTranslationPidController, xTranslationPidController, rotationPidController;
+    public double kP;
+    public double kI; 
+    public double kD;
+
+
+    public AutoPickupNote(){
+        swerve = Swerve.getInstance();
+        limelight = SkyLimelight.getInstance();
+        debouncer = new Debouncer(.5);
+
+        // creating yTranslationPidController and setting the toleance and setpoint
+        yTranslationPidController = new PIDController(0, 0, 0);
+        yTranslationPidController.setSetpoint(0);
+    
+        // creating xTranslationPidController and setting the toleance and setpoint
+        xTranslationPidController = new PIDController(0, 0, 0);
+        xTranslationPidController.setSetpoint(0);
+        
+        // creating rotationPidController and setting the toleance and setpoint
+        rotationPidController = new PIDController(0, 0, 0);
+        rotationPidController.setSetpoint(0);
+
+        // puts the value of P,I and D onto the SmartDashboard
+        // Will remove later
+        SmartDashboard.putNumber("P", 0);
+        SmartDashboard.putNumber("i", 0);
+        SmartDashboard.putNumber("d", 0);
+        //THESE VALUES WILL NEED TO BE MESSED WITH, 0 FOR NOW
+
+        addRequirements(swerve, intakeSubsystem);
+    }
+    private boolean NoteSeen(){
+        return limelight.hasTag();
+    }
+    private double RadiansToDegrees(double rads){
+        return rads * (180/Math.PI);
+    }
+    private double FixedDistanceCalc(){
+        double objectHeight = 0;
+        double cameraHeight = .5;
+        double yAngle = limelight.getY();
+        double skewAngle = limelight.getSkew();
```

Comment:
I'm not sure if the skew we get from the limelight will actually be useful in this case, or if it will matter. No need to change anything, just something to be aware of when we test this.

Reply:
removed skew and changed it to xValue